master/getting-started/mesos/installation/dc-os/custom.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Customizing Calico-DC/OS
+---
+
+The Calico-DC/OS Framework includes customization options which support
+more stable deployments when users
+
+#### Using Custom etcd
+
+We can adjust the framework to use an etcd cluster deployed outside of Universe.
```

Comment:
Just a general note we should be keeping in mind throughout this doc as well as the rest, I think we should be avoiding the first person plural (we).

It's better to use phrasing like "The Calico Universe framework can be configured to use an existing etcd cluster".